cmd/ak/cmd/projects/get.go

```diff
@@ -20,8 +23,8 @@ var getCmd = common.StandardCommand(&cobra.Command{
 			return err
 		}
 
-		if err := common.FailIfNotFound(cmd, "project", p.IsValid()); err != nil {
-			return err
+		if !p.IsValid() {
```

Comment:
I don't think this change is correct: the default behavior should be that all get commands return nothing if the requested entity isn't found (not even an error).

If the user specifies the `--fail` flag, then we return the "not found" error.

BTW - same thing for all list commands.

Reply:
> the default behavior should be that all get commands return nothing if the requested entity isn't found

Why? If the use explicitly asked for something that's not there - it should be an error.

Reply:
@tebeka right now this is how our cli is defined. even if used is asking for specific entity, cmd will fail silently if the entity isn't found, unless `--fail` flag is provided
@daabr - maybe I'm wrong, but IMO I introduced no changes. I retested with the new code and it produces the same results - silent fail if there is no such project, and error if `--fail` is specified